Dockerfile

```
@@ -1,143 +1,95 @@
 
-COPY --from=php_extension_installer --link /usr/bin/install-php-extensions /usr/local/bin/
-
 # persistent / runtime deps
-RUN apk add --no-cache \
-		acl \
-		fcgi \
-		file \
-		gettext \
-		git \
-	;
+RUN apt-get update && apt-get install --no-install-recommends -y \
```

Hadolint (2.12.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)